src/test/java/uk/gov/justice/services/test/domain/aggregate/MultipleObjectAggregate.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.services.test.domain.aggregate;
+
+import static uk.gov.justice.domain.aggregate.matcher.EventSwitcher.doNothing;
+import static uk.gov.justice.domain.aggregate.matcher.EventSwitcher.match;
+import static uk.gov.justice.domain.aggregate.matcher.EventSwitcher.when;
+
+import uk.gov.justice.domain.aggregate.Aggregate;
+import uk.gov.justice.services.test.domain.event.FirstEvent;
+import uk.gov.justice.services.test.domain.event.SecondEvent;
+import uk.gov.justice.services.test.domain.event.ThirdEvent;
+
+import java.util.stream.Stream;
+
+public class MultipleObjectAggregate implements Aggregate {
+
+    private boolean firstEventApplied;
+    private boolean secondEventApplied;
+
+    public Stream<Object> action() {
```

Comment:
might be useful to call this performAction so that the language makes sense in the test

Reply:
done